src/org/jetbrains/plugins/scala/codeInsight/intention/booleans/ReplaceEqualsOrEqualityInMethodCallExprIntention.scala

```diff
@@ -44,7 +46,12 @@ class ReplaceEqualsOrEqualityInMethodCallExprIntention extends PsiElementBaseInt
     false
   }
 
-  override def invoke(project: Project, editor: Editor, element: PsiElement) {
+  override def invoke(project: Project, editor: Editor, element: PsiElement): Unit = {
+    invoke_method_one(project, editor, element)
```

Comment:
I would rewrite this inspection with string interpolation instead of `StringBuilders`, I think it would be much cleaner.